README.md

```diff
@@ -783,31 +783,18 @@ This is a known issue:
 
 ### vim.call()
 
-`vim.call()` calls a Vimscript function. This can either be a built-in Vim function or a user function. Again, data types are converted back and forth from Lua to Vimscript.
+`vim.call()` is the less readable choice to call a Vimscript function. It should be avoided in favour of the equivalent `vim.fn.{function}()`.
```

Comment:
I feel like "less readable" is subjective

Reply:
true. What do you think about "more explicit, since it separates the function name and the arguments"?
Or should I also explain, how the function name is separate?